tasks/package.yml

```diff
@@ -7,13 +7,15 @@
   when: ansible_os_family == 'Debian'
 
 - name: Install the GPG key (Debian-like)
-  apt_key: file=/tmp/DUO-GPG-PUBLIC-KEY id=C1A07A85 state=present
+  apt_key:
+    url: https://duo.com/DUO-GPG-PUBLIC-KEY.asc
```

Comment:
The GPG key is already included in the role as files/DUO-GPG-PUBLIC-KEY. I checked and the keys are identical:

```sh
>>> md5sum ./files/DUO-GPG-PUBLIC-KEY*   
6dc7c866d16c7e045eedd4e432ae0085  ./files/DUO-GPG-PUBLIC-KEY
6dc7c866d16c7e045eedd4e432ae0085  ./files/DUO-GPG-PUBLIC-KEY.asc
```

We have environments with tightly controlled internet access and prefer to avoid internet dependencies like this.

Reply:
I'm happy to go along (ie, drop that change), its part of a larger set of changes I made updating key versions and so on when I first started to update the role. i think the other changes all fell out during rebase.

Reply:
Yeah, the outcome should be functionally the same for you either way.

Reply:
I've dropped this change from the branch